### PR TITLE
Support pegasus exporters

### PR DESCRIPTION
**Pull request by mht-sharma**
# What does this PR do?

Supports pegasus in exporters

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: tests/exporters/test_onnx_export.py [hunk 1]
```diff
@@ -107,11 +107,12 @@
     # Not using facebook/blenderbot-400M-distill because it takes too much time for testing, and might cause OOM.
     ("blenderbot", "facebook/blenderbot-90M"),
     # Not using google/bigbird-pegasus-large-arxiv because it takes too much time for testing.
-    ("bigbird-pegasus", "hf-internal-testing/tiny-random-bigbird_pegasus"),
+    # ("bigbird-pegasus", "hf-internal-testing/tiny-random-bigbird_pegasus"),
```
Comment (michaelbenayoun) [inline tests/exporters/test_onnx_export.py:110]: Why?

Comment (mht-sharma) [reply tests/exporters/test_onnx_export.py:110]: Was testing something would revert

Comment (mht-sharma) [reply tests/exporters/test_onnx_export.py]: Done

**Review by michaelbenayoun — APPROVED** [review]
LGTM!

**Review by fxmarty — APPROVED** [review]
LGTM! Sorry for the mess I merged https://github.com/huggingface/optimum/pull/575 , can you resolve conflicts and rerun the tests?

**Pull request merged**